### ISSUE TITLE
DomPDF has height issues in table row when the content exceeds one line.

### ISSUE THREAD

**Issue by pnt9948** [dompdf/dompdf#3123]
DomPDF has height issues in table row when the content exceeds one line.

**Comment by pnt9948** [comment]
Please find the following HTML for the reference.

`<div class="col1">
<img src="123.jpg" />
</div>
<div class="col2">
<table>
<tr><td>abc</td><td>abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz</td></tr>
</table>
</div>`

**Comment by bsweeney** [comment]
I'm not sure what I'm supposed to see with this sample but it appears to render as expected based on the HTML. I'm guessing there's some styling applied that may impact the layout?

**Comment by pnt9948** [comment]
Yes, we have applied the styling. We have given 35% width to col1 class and rest to col2 class. But the content in the col2 breaks. It's showing like this.

<table>
<tr><td>abc</td><td>abcdefghijklmnopqrstuvwxyz
<br /><br /><br />abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz</td></tr>
</table>

**Comment by bsweeney** [comment]
I'm not seeing that when I do a render using the following:

```
<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
  <style>
    .col1 {
      float: left;
      width: 35%;
    }
    .col2 {
      float: right;
      width: 65%;
    }
  </style>
</head>
<body>
  <div class="col1">
    <img src="https://placekitten.com/100/100" />
  </div>
  <div class="col2">
    <table>
      <tr>
        <td>abc</td>
        <td>abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz.abcdefghijklmnopqrstuvwxyz</td>
      </tr>
    </table>
  </div>
</body>
</html>
```

I'm going to need to see a more complete sample of your HTML + CSS. Also let us know which version of Dompdf you're using?

**Comment by pnt9948** [comment]
Hello @bsweeney

Please find attached css and html attached in this file. Please let us know if anything is required from our end.

Thanks,
Parth
[test.txt](https://github.com/dompdf/dompdf/files/10565159/test.txt)

**Comment by bsweeney** [comment]
OK, yes, definitely seeing that. I'll need to investigate more to determine what's going on there.

**Comment by pnt9948** [comment]
Okay thanks @bsweeney. Please let us know if anything is required from our end.

Thanks,
Parth
